lib/network/modules/LayoutEngine.js

```diff
@@ -30,6 +29,8 @@
  * The layout is a way to arrange the nodes in the view; this can be done
  * on non-hierarchical networks as well. The converse is also possible.
  */
+'use strict';
```

Comment:
Why move this

Reply:
No special reason, other than to keep the javascript code together. You'd rather have it on top?

Reply:
Fine as is, just curious